supported_images.md

```diff
@@ -0,0 +1,25 @@
+## Supported Image Versions
+
+The tables below list the supported SageMaker Distribution image versions. SageMaker will only release functionality and security fixes for images versions present in the tables. When you are building images or pulling images from the ECR repository, it is highly recommended for you to choose the supported image versions from the tables below.
+
+### CPU Images
+
+| Image Major Version | Image Minor Version |     ECR Image URI     |
```

Comment:
Just a minor comment: we can remove the last column from these tables. 

If needed, we can link the ECR Gallery once more in the narrative above, so that users know where to go to search for a particular image tag.

Reply:
I see, the reason why I added the column is for the users to know the exact image tags. For example when we add some appendix like `-cpu`, `-gpu`, the users will know what is the right image tag to access. Also we may have multiple patch versions for a minor version, like `1.8`, `1.8.1`, `1.8.2`, etc., we want to instruct the users to use `1.8` only which always points to the latest patch version.

Maybe instead of full URL, we can add a column for image tag. Something like `1.8-cpu` so the users can easily look up in our ECR repo